docs/http_api.rst

```diff
@@ -752,3 +752,16 @@ A status event can be categorised under one of the following levels:
     throttled.
   - ``down``: The component is not operational as a result of the issue
     described by the event.
+
+.. _health:
+
+Health
+^^^^^^
+
+.. http:get:: /health/
+
+   Provides HTTP GET access to test or verify the health of the system.
+
+   If the ``rabbitmq_management_interface`` config item is set it will also
+   query the RabbitMQ Management interface to check the health of each queue.
+   This is only available for RabbitMQ.
```

Comment:
I think the important thing here is just to add that a 200 is returned if the health is okay, and a 500 is returned if it's not. Maybe also an example of the JSON returned if the `rabbitmq_management_interface` config item is set, so that people can see the format of the response in case they want the details.